yt_dlp/extractor/zdf.py

```diff
@@ -136,18 +136,18 @@ def _extract_player(self, webpage, video_id, fatal=True):
 class ZDFIE(ZDFBaseIE):
     _VALID_URL = r'https?://www\.zdf\.de/(?:[^/]+/)*(?P<id>[^/?#&]+)\.html'
     _TESTS = [{
-        # Same as https://www.3sat.de/film/ab-18/10-wochen-sommer-108.html
-        'url': 'https://www.zdf.de/dokumentation/ab-18/10-wochen-sommer-102.html',
-        'md5': '0aff3e7bc72c8813f5e0fae333316a1d',
+        'url': 'https://www.zdf.de/nachrichten/heute-journal/heute-journal-vom-30-12-2021-100.html',
         'info_dict': {
-            'id': '141007_ab18_10wochensommer_film',
+            'id': '211230_sendung_hjo',
             'ext': 'mp4',
-            'title': 'Ab 18! - 10 Wochen Sommer',
-            'description': 'md5:8253f41dc99ce2c3ff892dac2d65fe26',
-            'duration': 2660,
-            'timestamp': 1608604200,
-            'upload_date': '20201222',
-        },
+            'description': 'md5:47dff85977bde9fb8cba9e9c9b929839',
+            'duration': 1890.0,
+            'upload_date': '20211230',
+            'chapters': [{'start_time': 0, 'end_time': 31.62, 'title': '<Untitled>'}, {'start_time': 31.62, 'end_time': 296.56, 'title': 'Omikron-Erkenntnisse aus Südafrika'}, {'start_time': 296.56, 'end_time': 636.54, 'title': 'Drosten: "Omikron könnte Impflücke schließen"'}, {'start_time': 636.54, 'end_time': 883.07, 'title': 'Nachrichten'}, {'start_time': 883.07, 'end_time': 1483.17, 'title': '2021: Der Jahresrückblick'}, {'start_time': 1483.17, 'end_time': 1890, 'title': 'Bye Bye Claus'}],
```

Comment:
```suggestion
            'chapters': list,
```
should be sufficient